spec/mas/cms/attribute_builder_spec.rb

```diff
@@ -26,6 +26,11 @@ module Mas::Cms::Repository::CMS
         expect(subject['description']).to eq(expected)
       end
 
+      it 'returns meta_title' do
+        expected = 'How to Manage Your Money – Beginner\'s Guide'
+        expect(subject['meta_title']).to eq(expected)
+      end
```

Comment:
Is it worth updating this fixture as well? https://github.com/moneyadviceservice/mas-cms-client/blob/892ed44377367a4b11a8d1514a82428a99b9de3c/spec/mas/cms/attribute_builder_spec.rb#L7

Reply:
Awesome then! @MarcoBgn